src/app/containers/ATIAnalytics/index.jsx

```diff
@@ -19,6 +19,7 @@ const ATIAnalytics = ({ data }) => {
     frontPage: buildFrontPageATIUrl,
     media: buildRadioATIUrl,
     MAP: buildCpsAssetPageATIUrl,
+    PGL: buildCpsAssetPageATIUrl,
```

Comment:
Wondering if we need to split the URL builders for the different CPS assets just like we did for the containers. e.g. `buildMapPageATIUrl`, `buildPglPageATIUrl` and `buildStyPageATIUrl`. Just in case there's separate logic needed for building them. Need to check with product if this is the case.

Reply:
As you say, I'd just check with product what the requirements are. If you get this issue refined with @lizcameron https://github.com/bbc/simorgh/issues/5230 hopefully you'll get the info you need.

Reply:
1 change to content type. For PGLs:
Content type = article-photo-gallery

Reply:
This is done!